### DISCUSSION TITLE
*** Important Branch Update ***

### DISCUSSION THREAD

**Discussion by Jyers** [Jyers/Marlin#1047] [General Discussion]
### Quick note as part of clearing the repo of the older **Extensible UI** branding in favor of the new **JyersUI** I have made a new default branch. If you have your own local repo of the firmware you'll either want to reclone it on the new branch or reset the remote upstream to the new branch.

**Comment by adam3654** [comment]
I'm a noob when it comes to GitHub, and I have made a bunch of little tweaks to a clone of your repo. Is there a way to save what changes I've made and apply it to the new branch?

**Reply by Jyers** [reply]
Definitely, the easiest way would probably be to checkout to the new branch and them merge from your previous one. If you need any help with that let me know :)

**Reply by adam3654** [reply]
Do I need to clone the new branch locally first, I just tried checking it out, but it only has the option to check out detached. 
When I checked it out I lost the workspace on the left side panel, I'm using the GitLens extension in VS code.

**Reply by Jyers** [reply]
It should be possible to do so without cloning the new branch first, but both approached should work fine.

**Reply by adam3654** [reply]
I was able to get it working, thanks for the help.

**Comment by HermanGaviriaMarin** [comment]
How does this affect normal everyday users who aren't too savvy with coding and GitHub. Will this affect those of us who normally just download the precompiled BIN firmware ?

**Reply by Jyers** [reply]
Nope, not at all :P Everything will be that same as usual, you can just grab new bins from the releases section

**Comment by smooreace** [comment]
There is no coding involved in compiling firmware. Just a little text
editing
